src/components/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import '../App.css';
+import Display from './display.component';
+import ButtonPanel from './buttonPanel.component';
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      result: 0,
+    };
+  }
```

Comment:
- Why are you setting the result to `0` here?, remember that this should be a functional component as mentioned on Pathwright, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/appFunctionalComponent.png), therefore you do not need the constructor part rather you should only render the `Display` and `ButtonPanel` components in a `div`